src/node.cc

```diff
@@ -2768,19 +2768,19 @@ void SetupProcessObject(Environment* env,
   // process.release
   Local<Object> release = Object::New(env->isolate());
   READONLY_PROPERTY(process, "release", release);
-  READONLY_PROPERTY(release, "name", OneByteString(env->isolate(), "io.js"));
+  READONLY_PROPERTY(release, "name", OneByteString(env->isolate(), "node.js"));
```

Comment:
I think I'd vote for `"node"` but it's a line-ball. Votes from other @nodejs/tsc on `process.release.name` value?